packages/workflow/src/documents.ts

```diff
@@ -70,57 +70,70 @@ export async function uploadCertificateAttachmentsToDocumentsStore<
   return certificateDetails
 }
 
+function isPresignedUrl(url: string) {
+  return url.includes(MINIO_URL)
+}
+
+function getMinioUri(url: string) {
+  const parsedUrl = new URL(url)
+  const filePath = parsedUrl.pathname
```

Comment:
Good solution 👍

Reply:
Could we move this function to attachments and call it something like 

```ts
function getCanonicalAttachmentURL(presignedOrCanonical: string) {
  if(presignedOrCanonical) {
    return <path name>
  }
  return presignedOrCanonical
}
```
please?